model/pluginconf.go

```diff
@@ -0,0 +1,34 @@
+package model
+
+// PluginConf holds information about the plugin
+type PluginConf struct {
+	ID       uint `gorm:"primary_key;AUTO_INCREMENT;index"`
+	UserID   uint
+	FileName string
+	Token    string `gorm:"unique_index"`
+	Enabled  bool
+	Config   []byte
+}
+
+// PluginConfExternal Model
+//
+// Holds information about a plugin instance for one user.
+//
+// swagger:model PluginConf
+type PluginConfExternal struct {
+	// The user name. For login.
+	//
+	// required: true
+	// example: P1234
+	Token string `binding:"required" json:"token" query:"token" form:"token"`
+	// The file name of the plugin.
+	//
+	// example: rss.so
+	// read only: true
+	// required: true
+	FileName string `json:"fileName" form:"fileName" query:"fileName"`
+	// Whether the plugin instance is enabled.
+	//
+	// example: true
+	Enabled bool `json:"enabled"`
```

Comment:
This should also be required as it is never unset because it is a primitive.

Reply:
Yeah right, thanks for your reminder. I'll read more about swagger docs about these. :)